play/src/front/Phaser/Entity/Area.ts

```diff
@@ -49,6 +56,15 @@ export class Area extends Phaser.GameObjects.Rectangle {
 
     destroy(fromScene?: boolean) {
         super.destroy(fromScene);
+
+        this.areaData.properties.forEach((property) => {
+            const validationResult = MatrixRoomPropertyData.safeParse(property);
```

Comment:
I think you could do simply something like: `if (property.type === 'matrixRoomPropertyData')`